server/server.go

```diff
@@ -47,11 +48,19 @@ type Server struct {
 
 	// If provided, enables the SAML integration.
 	ServiceProvider *saml.ServiceProvider
+
+	OauthRedirectURL *url.URL
 }
 
 func New(e *empire.Empire, options Options) *Server {
 	s := &Server{}
 
+	if options.OauthRedirectURL != "" {
+		parsedUrl, err := url.Parse(options.OauthRedirectURL)
+		if err == nil {
+			s.OauthRedirectURL = parsedUrl
+		}
+	}
```

Comment:
I think we should handle the error case here. This could result in very frustrating behavior where the url is set but invalid and emp v1 just ignores it. I'd do something like 

```suggestion
		parsedUrl, err := url.Parse(options.OauthRedirectURL)
		if err != nil {
		        // return some sort of appropriate error
		}
			
		s.OauthRedirectURL = parsedUrl
	}
```

Reply:
I think that'd be more idiomatic for go, following the `if err != nil` pattern.

Reply:
fixed